src/Pate/Monad.hs

```diff
@@ -555,13 +642,14 @@ checkSatisfiableWithoutBindings
   :: (sym ~ WE.ExprBuilder t st fs)
   => PT.Timeout
   -> sym
+  -> String
   -> IO (W4R.SatResult (W4G.GroundEvalFn t) ())
-  -> IO (W4R.SatResult (W4G.GroundEvalFn t) ())
-checkSatisfiableWithoutBindings timeout _sym doCheckSat =
+  -> EquivM sym arch (W4R.SatResult (W4G.GroundEvalFn t) ())
```

Comment:
I just noticed that this changed from `IO` to `EquivM`, but it doesn't look like anything in here uses that additional context?

Is this intentional or a forward looking change? I'm not opposed, but curious.

Reply:
No I think it made sense at some point but eventually all of the EquivM stuff eventually ended up outside the function. I'll just switch it back to IO for clarity.